multiapps-controller-persistence/pom.xml

```diff
@@ -198,5 +198,10 @@
             <groupId>org.eclipse.persistence</groupId>
             <artifactId>org.eclipse.persistence.extension</artifactId>
         </dependency>
+        <dependency>
+			<groupId>org.bouncycastle</groupId>
+			<artifactId>bcpkix-jdk15on</artifactId>
+			<version>1.70</version>
+		</dependency>
```

Comment:
Extract the dependency in the parent pom, make the version as a property and just refer it in this maven module

Reply:
Add it in parent pom.xml under `<dependencyManagement>` section and here just specify the dependency group id and artifact id without version.